src/material-experimental/mdc-snackbar/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary", "sass_library")
+load("//:packages.bzl", "CDK_TARGETS", "MATERIAL_TARGETS")
+load("//tools:defaults.bzl", "ng_module")
+
+ng_module(
+    name = "mdc-snackbar",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.spec.ts"],
+    ),
+    assets = [":snackbar_scss"] + glob(["**/*.html"]),
+    module_name = "@angular/material-experimental/mdc-snackbar",
+    deps = [
+        "@npm//material-components-web",
+    ] + CDK_TARGETS + MATERIAL_TARGETS,
```

Comment:
We shouldn't be doing this. This adds all possible build targets as dependencies which will cause very slow rebuilds. We should list out the dependencies individually.

Reply:
Done.